integration-cli/docker_api_attach_test.go

```diff
@@ -120,12 +121,12 @@ func (s *DockerSuite) TestPostContainersAttach(c *testing.T) {
 			}
 			assert.Equal(c, actual[0], fdMap[stream])
 		}
-		assert.Assert(c, is.DeepEqual(actual[lenHeader:], expected), "Attach didn't return the expected data from %s", stream)
+		assert.Assert(c, is.DeepEqual(actual[lenHeader:], expected), "Attach didn'c return the expected data from %s", stream)
```

Comment:
Looks like one find & replace too many 😅 

```suggestion
		assert.Assert(c, is.DeepEqual(actual[lenHeader:], expected), "Attach didn't return the expected data from %s", stream)
```